cogeo_mosaic_tiler/handlers/app.py

```diff
@@ -313,7 +314,7 @@ def _wmts(
         )
 
 
-params["cache_control"] = (os.environ.get("CACHE_CONTROL", None),)
+params["cache_control"] = os.environ.get("CACHE_CONTROL", None)
```

Comment:
Is this here that only the GET methods get it?

Reply:
yes, it's only used for the `tiles` and `point` routes, should we cache the others (wmts, tilejson, info) ?

Reply:
I think it's fine to cache the others, because returned results are cached including the query string. So there's not really a downside to caching all of them.